templates/vhost/_gssapi.epp

```diff
@@ -1,10 +1,14 @@
 <%|
   # https://github.com/gssapi/mod_auth_gssapi
+  Optional[Enum['On','Off']] $basicauth = undef,
   Optional[String[1]]        $credstore = undef,
   Optional[Enum['On','Off']] $sslonly   = undef,
   Optional[Enum['On','Off']] $localname = undef,
 |%>
-# mod_auth_gssapi configuration
+    # mod_auth_gssapi configuration
```

Comment:
I am wondering about this and whitespace. It looks correct, but have you manually verified this looks sane?

Reply:
Yep, it does:
```
  <Location "/somewhere">
    <RequireAll>
      Require user abc
    </RequireAll>
    AuthType GSSAPI
    AuthName "My ID"
      
    # mod_auth_gssapi configuration
    GssapiBasicAuth On
    GssapiSSLonly Off
    GssapiLocalName On
    GssapiCredStore keytab:/etc/httpd/conf/http.keytab
  </Location>
```